samples/HttpServer_FirmwareUpload/app/SignedRbootOutputStream.cpp

```diff
@@ -35,23 +34,21 @@ size_t SignedRbootOutputStream::write(const uint8_t* data, size_t size)
 				if(unexpectedMagic || unexpectedLoadAddress) {
 					setError(unexpectedLoadAddress ? "Unexpected load address. Try image for other slot."
 												   : "Invalid image received.");
-					return 0;
```

Comment:
If there is an error just return 0; This way the request will be terminated earlier.